src/radix_connect/mobile/session/session.rs

```diff
@@ -0,0 +1,62 @@
+use super::session_id::SessionID;
+use super::session_origin::SessionOrigin;
+use crate::prelude::*;
+
+#[derive(Debug, PartialEq)]
+pub struct Session {
+    pub id: SessionID,
+    pub origin: SessionOrigin,
+    pub encryption_key: Exactly32Bytes,
```

Comment:
Ah, ok you can for now change to `Entropy32Bytes` (might wanna rename it later) which implements `Zeroize` and then you can add `Zeroize` to the derive.

Reply:
dumb question I guess - Is `Entropy32Bytes` applicable here also? I mean this would be later a key derived from DH key agreement, isn't Entropy32Bytes about having a key derived from mnemonic?